go.mod

```diff
@@ -42,6 +42,6 @@ require (
 	sigs.k8s.io/yaml v0.0.0-20181102190223-fd68e9863619 // indirect
 )
 
-replace github.com/bronze1man/goStrongswanVici => github.com/mangelajo/goStrongswanVici v0.0.0-20190701121157-9a5ae4453bda
+replace github.com/bronze1man/goStrongswanVici => github.com/mangelajo/goStrongswanVici v0.0.0-20190701121157-73b0716138ff
```

Comment:
(nv) for future changes it might be wise to update the version date as well

Reply:
I will re-point this to the upstream goStrongswanVici now again, since they merged all our patches :-)

Reply:
Then we can point to a specific version when they tag it.

Reply:
Ok, I pointed to strongswanVici master from the u/s repo.